Packs/Phishing/Playbooks/Process_Email_-_Generic_v2.yml

```diff
@@ -2789,10 +2789,11 @@ tasks:
       id: 754742b3-8a98-45b3-8d02-a850f90d83fd
       version: -1
       name: Upload email attachments to layout
-      description: Uploads file attachments of the email to the incident layout.
-      script: '|||core-api-multipart'
+      description: Copies a file from this incident to the specified incident. The
+        file is recorded as an entry in the specified incident’s War Room.
+      scriptName: UploadFile
       type: regular
-      iscommand: true
+      iscommand: false
```

Comment:
Is an automation does not qualify as command in the PB anymore?

Reply:
[Example](https://github.com/demisto/content/blob/master/Packs/DeveloperTools/TestPlaybooks/playbook-EDL_Performance_Test_-_Concurrency.yml#L436), I believe if the task has an automation, the **iscommand** should be false.